program/engagement/transition.md

```diff
@@ -0,0 +1,79 @@
+- title: Step 2 – Change Strategy
+  body: | 
+    A [change strategy]() will outline the way in which the change to modern desktop will be managed. This is so that the value of the Blueprint c an be maximised, and risks to successful adoption minimised. The Change Strategy will determine a vision for change, who will be communicated with and when (i.e a [Communication Plan], a [Skills and Training Needs Assessment] or learning strategy, and a clear [Benefits] register or strategy.
+- title: Step 3 – Change Plan
+  body: | 
+    A [change plan]() will outline the concrete steps to be enacted in the short-term. 
+- title: Step 4 – Enact the change
+  body: | 
+    Change will be implemented in this step based on the change plan.
+- title: Step 5 – Security Assessment
+  body: | 
+    A security assessment will be conducted to understand if the modern desktop services, applications, and data are secure and compliant with the [Information Security Manual (ISM)]() and other cyber security policies.
+- title: Step 5 – Security Assessment
+  body: | 
+    A security assessment will be conducted to understand if the modern desktop services, applications, and data are secure and compliant with the [Information Security Manual (ISM)]() and other cyber security policies.
+- title: Step 6 - Training and Communications
+  body: | 
+    During this step, staff will be supported in upskilling to use modern desktop services. This may involve e-Learning, face to face training, and peer training. Communications material will also be developed to support the change.
+- title: Next steps
+  body: | 
+    * After Transition, the customer agency must consider its next steps. These may include:  
+    * Undergoing further change planning in [Transition]() again. 
+    * Undergoing technology build and deployment in [Deploy]() 
+    * Move to [Adoption]() and start to realise the benefits of a modern desktop. 
+---
+
+During transition, agencies adopt the business change required to work in a modern, secure M365 desktop environment. Benefits are realised iteratively, and the necessary improvements are made to enable this. This includes adapting the business processes to suit the new technology landscape. The customer is independent, empowered, and able to drive its own outcomes. Technology support is provided by the vendor. Lessons learnt and knowledge are shared through a WoG community of practice led by the DTA.   
+
+Transition can happen before or after [Deploy]() (product management). The transition phase may also be implemented iteratively, alongside Deploy. 
+
+The DTA may co-fund transition activities with agencies.  
+
+## Roles 
+
+* The DTA
+* The customer agency
+* Organisational Change Management (OCM) partner (optional) 
+
+## Tools, Templates and Guidance 
+
+* [Change readiness assessment]() 
+* [Change strategy]() 
+* [Change plan]() 
+* [Benefits register]() 
+* [Training and skills analysis]() 
+* [Closure letter]() 
+
+## Enabling Services 
```

Comment:
Extra line break needed

Reply:
Not between bullets, I meant there needs to be a blank line before the start of the bullet list, eg: 

```
## Enabling Services

* The community...
```

Reply:
OIC! I'll update that now